src/shell/Shell.cpp

```diff
@@ -571,7 +575,15 @@ static void executeInvokeAction(wabt::InvokeAction* action, Walrus::Function* fn
             RELEASE_ASSERT(data->fn->functionType()->result().size() == data->expectedResult.size());
             // compare result
             for (size_t i = 0; i < result.size(); i++) {
-                RELEASE_ASSERT(equals(result[i], data->expectedResult[i]));
+                if (!equals(result[i], data->expectedResult[i])) {
+                    printf("Assert FAILED (line: %d) ", data->action->loc.line);
```

Comment:
Are you going to replace all `RELEASE_ASSERT` statements by this approach?
If so, please consider all `RELEASE_ASSERT` cases in this patch

Reply:
Good point!